b2share/modules/access/permissions.py

```diff
@@ -68,7 +68,7 @@ def authenticated_only(*args, **kwargs):
    and authenticated users.
 """
 
-class StrictDynamicPermission(DynamicPermission):
+class StrictDynamicPermission(Permission):
```

Comment:
DynamicPermission was enabling by default (See https://github.com/inveniosoftware/invenio-access/pull/162/files#diff-e21fe482cea0afb14433ca14c0b7702d85e8ff49fa04a7c23d2f6b3e48ad82d3L186-L192), Permission is rejecting by default (https://github.com/inveniosoftware/invenio-access/blob/87dde7f6b43bda41ec2115ef4a358442d8136703/invenio_access/permissions.py#L85).

There is a commit which brings DynamicPermission to B2SHARE code and removes `from invenio_access import DynamicPermission`: https://github.com/hjhsalo/b2share/commit/c94ddfbcb87f713c5daa8227f708ef7ac81fd306